.github/workflows/check-headers.yml

```diff
@@ -0,0 +1,32 @@
+name: Headers
+
+on:
+  pull_request:
+
+permissions: {}
+
+jobs:
+  check-headers:
+    name: Check headers
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
```

Comment:
Is this still necessary given that the header check is now moved outside of `validatePullRequest` and being done automatically on each PR?

Reply:
I don't think we need it for the check, good point.

Reply:
Cool, I just removed it and force pushed PR